hrjobcontract/CRM/Hrjobcontract/BAO/HRJobContractRevision.php

```diff
@@ -315,19 +315,6 @@ private static function buildEntityQueryArray($entity, $revision, $query) {
         ";
         break;
 
-      case 'pension':
```

Comment:
No need, this is handled in front-end, just as it's done for health and life insurance providers.  The only place where this label was used was on contract summary page, and now that it's handled by obtining the contact, is unnecessary.